Roles/Crewmate/Merchant.cs

```diff
@@ -132,10 +137,16 @@ public static void OnTaskFinished(PlayerControl player)
             {
                 return;
             }
+            if (addons.Count <= 0)
```

Comment:
Here you can use `!addons.Any()` this is more performant than `.Count <= 0`

Reply:
I see